### PR TITLE
Fix svbar when black

### PR DESCRIPTION
**Pull request by lack**

### CODE REVIEW

File: app/src/main/kotlin/com/github/oryanmat/trellowidget/util/color/ColorPreference.kt [hunk 1]
```diff
@@ -30,7 +30,7 @@ class ColorPreference @JvmOverloads constructor(
             addOpacityBar(view.opacitybar)
             color = getPersistedInt(DEFAULT_VALUE)
             oldCenterColor = getPersistedInt(DEFAULT_VALUE)
-            onColorChangedListener = ColorPicker.OnColorChangedListener { color = it }
+            onColorChangedListener = ColorPicker.OnColorChangedListener { this@ColorPreference.color = it }
```
Comment (oryanm) [inline app/src/main/kotlin/com/github/oryanmat/trellowidget/util/color/ColorPreference.kt:42]: nice catch 👍

File: app/src/main/kotlin/com/github/oryanmat/trellowidget/util/color/ColorPreference.kt [hunk 2]
```diff
@@ -23,6 +23,15 @@ class ColorPreference @JvmOverloads constructor(
         dialogLayoutResource = R.layout.color_chooser
     }
 
+    fun repairSVBar(view: View, colorToFix: Int) {
+        // This is a work-around for a defect in holocolorpicker when using black with an SVBar
```
Comment (oryanm) [inline app/src/main/kotlin/com/github/oryanmat/trellowidget/util/color/ColorPreference.kt:27]: can we try to get holopicker to fix this?

Comment (oryanm) [reply app/src/main/kotlin/com/github/oryanmat/trellowidget/util/color/ColorPreference.kt:27]: I guess he's not been very active recently

Comment (lack) [reply app/src/main/kotlin/com/github/oryanmat/trellowidget/util/color/ColorPreference.kt:27]: I will try to submit a pull request to that repo as well, but figure this workaround is pretty lightweight until (if) that fix goes in there.

**Pull request merged**